### PR TITLE
Fallback to bytes32 name and symbol if first read fails

### PR DESCRIPTION
**Pull request by sunify**

### CODE REVIEW

File: src/config/mainnet/config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "title": "LeapDAO Mainnet",
   "description": "Plasma network running transfer-only MoreVP governed by LeapDAO",
   "consensus": "poa",
-  "nodes": ["wss://mainnet-node1.leapdao.org:1443"],
+  "nodes": ["https://mainnet-node1.leapdao.org"],
```
Comment (troggy) [inline src/config/mainnet/config.json]: why this change is needed?

Comment (sunify) [reply src/config/mainnet/config.json]: By accident
